packages/block-library/src/cover/index.js

```diff
@@ -23,6 +23,12 @@ export const settings = {
 		'Add an image or video with a text overlay — great for headers.'
 	),
 	icon,
+	supports: {
```

Comment:
This `supports` has been moved to the `block.json` file for most blocks.

Reply:
Oh, thank you! I must have missed that. Will made an adjustment